common/reviews/api/rush-lib.api.md

```diff
@@ -144,6 +145,22 @@ export class IndividualVersionPolicy extends VersionPolicy {
     validate(versionString: string, packageName: string): void;
 }
 
+// @public
+export interface IPnpmOptionsJson {
+    pnpmStore?: IPnpmStoreOptions;
+    resolutionStrategy?: ResolutionStrategy;
+    strictPeerDependencies?: boolean;
+}
+
+// @public
+export interface IPnpmOptionsRootPaths {
+    commonTempFolder: string;
+    repoRoot: string;
+}
+
+// @public
+export type IPnpmStoreOptions = 'local' | 'global';
```

Comment:
This isn't an interface, so its name shouldn't start with an `I`

Reply:
Fixed